icekit/articles/abstract_models.py

```diff
@@ -1,12 +1,12 @@
-from icekit.abstract_models import FluentFieldsMixin
+from urlparse import urljoin
+
+from django.template.response import TemplateResponse
+
 from icekit.publishing.models import PublishingModel
-from django.db import models
+from polymorphic.models import PolymorphicModel
```

Comment:
Confused about why this particular case warrants specifying a version in setup.py. Surely by the same argument every requirement in setup.py should have a minimum supported version? And new installations of ICEkit won't include polymorphic 0.8, right?